### PR TITLE
Add Apex to supported languages

### PR DESCRIPTION
**Pull request by dschach**
Add Apex (Salesforce) language

### Changes
Include Apex in table

### Checklist
- [x] Added markup tests, or they don't apply here because... All tests are in the repo
- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

**Comment by dschach** [comment]
Apex is now also on NPM (and all the CDN services automatically). 
https://www.npmjs.com/package/highlightjs-apex

**Review by joshgoebel — CHANGES_REQUESTED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 Grammars:
 
+- Added 3rd party Apex grammar to SUPPORTED_LANGUAGES (#3546) [David Schach](https://github.com/dschach)
```
[the remark this answers is not in the document]
Comment (dschach) [reply CHANGES.md]: That was already fixed, I believe.

File: SUPPORTED_LANGUAGES.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ The table below shows the full list of languages (and corresponding classes/alia
 | ABNF                    | abnf                   |         |
 | Access logs             | accesslog              |         |
 | Ada                     | ada                    |         |
+| Apex                    | lightning              |[highlightjs-apex](https://github.com/highlightjs/highlightjs-apex/)  |
```
[the remark this answers is not in the document]
Comment (dschach) [reply SUPPORTED_LANGUAGES.md]: Done

**Comment by joshgoebel** [comment]
Please do not reformat the entire markdown file, just add the single line you need (so the diff is clear on what's changed), it's doesn't matter whether the brackets are perfect or not in the ASCII form just so long as it render fine on GitHub...

**Comment by dschach** [comment]
Fixed. Ready for review.

**Review by joshgoebel — APPROVED** [review]
Thanks so much! Perfect!

**Pull request merged**